sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatClientBuilder.java

```diff
@@ -141,6 +142,21 @@ public ChatClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+        /**
+     * Sets the {@link RetryPolicy} that is used when each request is sent.
+     * <p>
+     * The default retry policy will be used in the pipeline, if not provided.
+     *
+     * @param retryPolicy User's retry policy applied to each request.
+     * @return The updated {@link ChatClientBuilder} object.
+     * @throws NullPointerException If the specified {@code retryPolicy} is null.
+     */
+    public ChatClientBuilder retryPolicy(RetryPolicy retryPolicy) {
+        this.retryPolicy = Objects.requireNonNull(retryPolicy, "The retry policy cannot be bull");
```

Comment:
Quick change :), from bull to null.

Reply:
haha great catch! I dont know how that got there :D